clm_data/create_artifact.jl

```diff
@@ -2,8 +2,17 @@ using Downloads
 
 using ClimaArtifactsHelper
 
-const FILE_URL = "https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/surfdata_map/surfdata_0.9x1.25_hist_16pfts_nourb_CMIP6_simyrPtVg_c181114.nc"
-const FILE_PATH = "surfdata_0.9x1.25_hist_16pfts_nourb_CMIP6_simyrPtVg_c181114.nc"
+const FILE_URLS = [
+    "https://svn-ccsm-inputdata.cgd.ucar.edu/trunk/inputdata/lnd/clm2/surfdata_map/surfdata_0.9x1.25_16pfts__CMIP6_simyr2000_c170616.nc",
```

Comment:
This file has changed, is it equivalent to what we had before?

Reply:
The file in /groups/esm/ClimaArtifacts/artifacts/$artifact_name was not the same as the file the old script downloaded. This caused the dominant_pft.py script to fail. I changed the create_artifact.jl script to download that same file that is on calhpc, which allows dominant_pft.py to run.